libnavui-base/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
hmmm, I believe it should live at `libnavui-resources`(not `libnavui-voice`), because it's the common resource, doesn't it? name `mapbox_navigation_view_color_primary` looks like a widely used

Reply:
You are correct that `values/colors.xml` and `values-night/colors`. xml is already defined in `libnavui-resources`. But looking at the colors you have defined here, they are not using the colors from global `libnavui-resources` module as you are hardcoding them here. So `colors.xml` inside `libnavui-voice` is just fine but let's make sure that we use the colors from `libnavui-resources` if no separate colors are needed. If you do need to define different colors, then please add `values-night/colors`. I hope that makes sense.

I am not sure what the `primary` and `secondary` colors mean here as the name are not self explanatory, but the FAB background color should use `colorSurface` and the icon on top of FAB should use `colorOnSurface`. 

Also let's not use the name `navigation_view` in colors because it was in `1.x` and not in `2.0`. Please look at https://github.com/mapbox/mapbox-navigation-android/blob/main/docs/resources-naming-convention.md#colors which defines how to name `color` resources.

cc @RingerJK